### PR TITLE
PHP - Date Functions - Terms - date_diff

### PR DESCRIPTION
**Pull request by bitsalt**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by bitsalt** [comment]
After going through the checklist, I see that I need to make a branch on my fork specific to this change.

**Comment by SSwiniarski** [comment]
@bitsalt You can make a new PR on a new branch, or we can review this as is, it just limits you to not submitting any more PRs until this one is merged. Just remember going forward not to commit your changes to the `main` branch.

**Comment by bitsalt** [comment]
I'll leave this one. It's my first contribution, so I'd rather get a feel for the entire workflow before taking another. Thanks.

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (SSwiniarski) [inline content/php/concepts/date-functions/terms/date-diff/date-diff.md:63]: Since this is a codebyte, why don't we return the current date here rather than a literal value.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@bitsalt This looks good. On review I only had one comment.

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (bitsalt) [reply content/php/concepts/date-functions/terms/date-diff/date-diff.md]: I changed $dateNow to actually use "now"

**Review by SSwiniarski — APPROVED** [review]
@bitsalt looks good for a second review.

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-diff/date-diff.md:28]:
```suggestion
The `$dateOne` and `$dateTwo` parameters are `DateTime` objects as returned by [`date_create()`](https://www.codecademy.com/resources/docs/php/date-functions/date-create). Because the returned value is a `DateInterval` object, [`date_format()`](https://www.codecademy.com/resources/docs/php/date-functions/date-format) cannot be used to format the result. The `DateInterval::format()` function can be used for formatting using the characters below. The formatting characters must be prefixed with a percent sign (`%`).
```

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-diff/date-diff.md:50]: Generally, if there is only one example it is a standard code block, not a codebyte. Please revise by adding another example, if you think there is another example you can add that would be valuable, or change this to a standard example with an output ```shell block.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@bitsalt Just a couple of additional comments. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (bitsalt) [reply content/php/concepts/date-functions/terms/date-diff/date-diff.md]: Backticks added to the percent sign as requested.

**Comment by bitsalt** [comment]
@caupolicandiaz, changes are complete. Thanks.

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-diff/date-diff.md:54]:
```suggestion
```php
```

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-diff/date-diff.md:62]: this code block should be followed by a line such as: 
"This code results in the following output:"

Followed by the output inside a ```shell block

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-diff/date-diff.md:28]:
```suggestion
The `$dateOne` and `$dateTwo` parameters are `DateTime` objects as returned by [`date_create()`](https://www.codecademy.com/resources/docs/php/date-functions/date-create). Because the returned value is a `DateInterval` object, [`date_format()`](https://www.codecademy.com/resources/docs/php/date-functions/date-format) cannot be used to format the result. The `DateInterval::format()` function can be used for formatting using the characters below. The formatting characters must be prefixed with a percent sign (`%`).
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@bitsalt A couple more changes to be made. Thanks

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (bitsalt) [reply content/php/concepts/date-functions/terms/date-diff/date-diff.md]: Added backticks to linked functions.

File: content/php/concepts/date-functions/terms/date-diff/date-diff.md
Comment (bitsalt) [reply content/php/concepts/date-functions/terms/date-diff/date-diff.md:62]: I added a block for example output.

**Review by caupolicandiaz — APPROVED** [review]
@bitsalt @SSwiniarski This is good to go

**Comment by bitsalt** [comment]
CLA signed. Sorry for the delay. I've been traveling.

**Comment by SSwiniarski** [comment]
@bitsalt Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/date-functions/date-diff

image

**Pull request merged**
